(286869@main): <details> disclosure triangle is now tiny with `system-ui`

vitorroriz opened the pull request:
#### 4593e7d6b65aa56d69fad5312d902a4ba12e5c38
<pre>
(286869@main): &lt;details&gt; disclosure triangle is now tiny with `system-ui`
<a href="https://bugs.webkit.org/show_bug.cgi?id=285480">https://bugs.webkit.org/show_bug.cgi?id=285480</a>
<a href="https://rdar.apple.com/141769356">rdar://141769356</a>

Reviewed by Tim Horton and Tim Nguyen.

On 280433@main when fixing the symbols according to direction input,  we have aligned
the symbol used for each direction. We have chosen to align them with the small variant as
specification [1] let this open and there is no consistency among different engines for
different directions.

Here we are changing it to align to the bigger variant of the disclosure triangles as it improves
readability.

* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-at-rule/disclosure-styles-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-at-rule/disclosure-styles-ref.html:
* LayoutTests/platform/glib/fast/css-generated-content/details-summary-before-after-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-details-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-details-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-1-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-10-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-10-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-2-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-3-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-3-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-4-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-4-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-5-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-5-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-6-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-6-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-7-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-7-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-8-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-8-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-9-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-9-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-marker-style-expected.txt:
* LayoutTests/platform/glib/fast/html/details-marker-style-mixed-expected.txt:
* LayoutTests/platform/glib/fast/html/details-nested-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-nested-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-no-summary1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-no-summary2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-no-summary3-expected.txt:
* LayoutTests/platform/glib/fast/html/details-open1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-open3-expected.txt:
* LayoutTests/platform/glib/fast/html/details-open5-expected.txt:
* LayoutTests/platform/glib/fast/html/details-open6-expected.txt:
* LayoutTests/platform/glib/fast/html/details-position-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-1-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-2-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-3-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-3-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-4-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-4-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-5-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-5-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-6-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-6-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-writing-mode-expected.txt:
* LayoutTests/platform/glib/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-open2-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-open4-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-open2-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-open4-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-details-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-details-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-1-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-10-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-10-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-2-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-3-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-3-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-4-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-4-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-5-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-5-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-6-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-6-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-7-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-7-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-8-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-8-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-9-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-9-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-marker-style-expected.txt:
* LayoutTests/platform/ios/fast/html/details-marker-style-mixed-expected.txt:
* LayoutTests/platform/ios/fast/html/details-nested-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-nested-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-no-summary1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-no-summary2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-no-summary3-expected.txt:
* LayoutTests/platform/ios/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open3-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open4-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open5-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open6-expected.txt:
* LayoutTests/platform/ios/fast/html/details-position-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-1-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-2-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-3-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-3-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-4-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-4-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-5-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-5-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-6-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-6-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/ios/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/ios/fast/html/details-writing-mode-expected.txt:
* LayoutTests/platform/ios/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-no-summary4-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-replace-summary-child-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk2/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk2/fast/html/details-replace-summary-child-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk2/fast/html/details-replace-text-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia/fast/html/details-no-summary4-expected.txt: Renamed from LayoutTests/platform/mac-sequoia-wk2/fast/html/details-no-summary4-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-open-javascript-expected.txt: Renamed from LayoutTests/platform/mac-sequoia-wk2/fast/html/details-open-javascript-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-open2-expected.txt: Renamed from LayoutTests/platform/mac-sequoia-wk2/fast/html/details-open2-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-open4-expected.txt: Renamed from LayoutTests/platform/mac-sonoma-wk1/fast/html/details-open4-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-replace-summary-child-expected.txt: Renamed from LayoutTests/platform/mac-sonoma-wk1/fast/html/details-replace-summary-child-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-replace-text-expected.txt: Renamed from LayoutTests/platform/mac-sonoma-wk1/fast/html/details-replace-text-expected.txt.
* LayoutTests/platform/mac-sonoma-wk1/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk1/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma/fast/html/details-marker-style-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/fast/html/details-marker-style-mixed-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/fast/html/details-no-summary2-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-no-summary2-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-no-summary4-expected.txt: Renamed from LayoutTests/platform/mac-sonoma-wk1/fast/html/details-no-summary4-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-remove-summary-1-and-click-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-remove-summary-1-and-click-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-remove-summary-4-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-remove-summary-4-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-remove-summary-child-2-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-remove-summary-child-2-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-replace-text-expected.txt: Renamed from LayoutTests/platform/mac-sequoia-wk1/fast/html/details-replace-text-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-writing-mode-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-writing-mode-mixed-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-writing-mode-mixed-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-writing-mode-mixed-expected.txt.
* LayoutTests/platform/mac-wk1/fast/html/details-no-summary4-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-replace-summary-child-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-replace-text-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/text-field-input-types-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-no-summary4-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-replace-summary-child-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-replace-text-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/text-field-input-types-expected.txt: Removed.
* LayoutTests/platform/mac/fast/css-generated-content/details-summary-before-after-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-details-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-details-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-1-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-10-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-10-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-2-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-3-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-3-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-4-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-5-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-5-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-6-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-6-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-7-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-7-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-8-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-8-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-9-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-9-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-marker-style-expected.txt:
* LayoutTests/platform/mac/fast/html/details-marker-style-mixed-expected.txt:
* LayoutTests/platform/mac/fast/html/details-nested-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-nested-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary3-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open3-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open5-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open6-expected.txt:
* LayoutTests/platform/mac/fast/html/details-position-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-1-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-2-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-3-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-3-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-4-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-5-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-5-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-6-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-6-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/mac/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/mac/fast/html/details-writing-mode-expected.txt:
* LayoutTests/platform/mac/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-open2-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-open4-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-replace-text-expected.txt:
* Source/WTF/wtf/unicode/CharacterNames.h:
* Source/WebCore/css/CSSCounterStyle.cpp:
(WebCore::CSSCounterStyle::counterForSystemDisclosureClosed):
(WebCore::CSSCounterStyle::counterForSystemDisclosureOpen):

Canonical link: <a href="https://commits.webkit.org/299530@main">https://commits.webkit.org/299530@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/deb5ddce7b8fe24f09f62bf4ae017b190a4df06d

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/119299 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/38986 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/29641 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/125537 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/71369 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/16e2068a-4a2f-4a33-b7d5-d602d4af838c) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/121176 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/39683 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/47567 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/90647 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/59990 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/dc561fbd-8117-4d3f-be5e-2295ba3b704f) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/122251 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/31660 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/106964 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/71072 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/b50573a1-fa01-4ce7-b448-9567f477454e) 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/30702 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/25074 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/69186 "Built successfully") | | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/111421 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/101112 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/25264 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/128544 "Built successfully") | | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/117815 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/46217 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/34960 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/99218 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/46582 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/103163 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/98996 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25162 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/44454 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/22468 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/42784 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/46080 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/51780 "Built successfully") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/146515 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/45545 "Built successfully") | | [❌ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/37669 "Found 1 new JSC binary failure: testapi, Found 18612 new JSC stress test failures: ChakraCore.yaml/ChakraCore/test/Array/array_slice.js.default, ChakraCore.yaml/ChakraCore/test/Array/push2.js.default, ChakraCore.yaml/ChakraCore/test/Array/reverse1.js.default, ChakraCore.yaml/ChakraCore/test/Array/toLocaleString.js.default, ChakraCore.yaml/ChakraCore/test/Bugs/bug215238.shrua-2.js.default, ChakraCore.yaml/ChakraCore/test/ControlFlow/forInPrimitive.js.default, ChakraCore.yaml/ChakraCore/test/Date/date_cache_bug.js.default, ChakraCore.yaml/ChakraCore/test/Error/CallNonFunction.js.default, ChakraCore.yaml/ChakraCore/test/Error/stack2.js.default, ChakraCore.yaml/ChakraCore/test/Function/FuncBody.bug227901.js.default ... (failure)") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/48895 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/47232 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on Source/WTF/wtf/unicode/CharacterNames.h:
```diff
@@ -46,8 +46,12 @@ constexpr UChar arabicPercentSign = 0x066A;
+constexpr UChar blackRightPointingTriangle = 0x25B6;
 constexpr UChar blackSquare = 0x25A0;
+constexpr UChar blackDownPointingSmallTriangle = 0x25BE;
+constexpr UChar blackDownPointingTriangle = 0x25BC;
```
This needs to be alpha sorted

Ahmad-S792 replied:
I just did it today.. LOL!

nt1m commented on Source/WTF/wtf/unicode/CharacterNames.h:
```diff
@@ -45,8 +45,11 @@ constexpr UChar arabicIndicPerTenThousandSign = 0x060A;
+constexpr UChar blackRightPointingTriangle = 0x25B6;
 constexpr UChar blackSquare = 0x25A0;
 constexpr UChar blackUpPointingSmallTriangle = 0x25B4;
 constexpr UChar blackUpPointingTriangle = 0x25B2;
```
I just realized we can remove the small variants from this file now that we no longer use them

nt1m commented on Source/WTF/wtf/unicode/CharacterNames.h:
```diff
@@ -185,8 +188,11 @@ using WTF::Unicode::arabicIndicPerTenThousandSign;
 using WTF::Unicode::arabicPercentSign;
 using WTF::Unicode::blackCircle;
 using WTF::Unicode::blackDownPointingSmallTriangle;
+using WTF::Unicode::blackDownPointingTriangle;
 using WTF::Unicode::blackLeftPointingSmallTriangle;
+using WTF::Unicode::blackLeftPointingTriangle;
 using WTF::Unicode::blackRightPointingSmallTriangle;
+using WTF::Unicode::blackRightPointingTriangle;
 using WTF::Unicode::blackSquare;
 using WTF::Unicode::blackUpPointingSmallTriangle;
 using WTF::Unicode::blackUpPointingTriangle;
```
ditto here

vitorroriz committed:
(286869@main): <details> disclosure triangle is now tiny with `system-ui`
https://bugs.webkit.org/show_bug.cgi?id=285480
rdar://141769356

Reviewed by Tim Horton and Tim Nguyen.

On 280433@main when fixing the symbols according to direction input,  we have aligned
the symbol used for each direction. We have chosen to align them with the small variant as
specification [1] let this open and there is no consistency among different engines for
different directions.

Here we are changing it to align to the bigger variant of the disclosure triangles as it improves
readability.

* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-at-rule/disclosure-styles-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/counter-style-at-rule/disclosure-styles-ref.html:
* LayoutTests/platform/glib/fast/css-generated-content/details-summary-before-after-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-details-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-details-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-1-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-10-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-10-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-2-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-3-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-3-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-4-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-4-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-5-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-5-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-6-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-6-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-7-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-7-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-8-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-8-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-9-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-9-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-add-summary-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-marker-style-expected.txt:
* LayoutTests/platform/glib/fast/html/details-marker-style-mixed-expected.txt:
* LayoutTests/platform/glib/fast/html/details-nested-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-nested-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-no-summary1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-no-summary2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-no-summary3-expected.txt:
* LayoutTests/platform/glib/fast/html/details-open1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-open3-expected.txt:
* LayoutTests/platform/glib/fast/html/details-open5-expected.txt:
* LayoutTests/platform/glib/fast/html/details-open6-expected.txt:
* LayoutTests/platform/glib/fast/html/details-position-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-1-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-2-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-3-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-3-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-4-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-4-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-5-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-5-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-6-and-click-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-6-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-child-1-expected.txt:
* LayoutTests/platform/glib/fast/html/details-remove-summary-child-2-expected.txt:
* LayoutTests/platform/glib/fast/html/details-writing-mode-expected.txt:
* LayoutTests/platform/glib/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-open2-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-open4-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/gtk/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-open2-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-open4-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/ios-18/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-details-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-details-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-1-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-10-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-10-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-2-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-3-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-3-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-4-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-4-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-5-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-5-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-6-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-6-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-7-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-7-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-8-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-8-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-9-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-9-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-add-summary-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-marker-style-expected.txt:
* LayoutTests/platform/ios/fast/html/details-marker-style-mixed-expected.txt:
* LayoutTests/platform/ios/fast/html/details-nested-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-nested-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-no-summary1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-no-summary2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-no-summary3-expected.txt:
* LayoutTests/platform/ios/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open3-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open4-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open5-expected.txt:
* LayoutTests/platform/ios/fast/html/details-open6-expected.txt:
* LayoutTests/platform/ios/fast/html/details-position-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-1-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-2-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-3-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-3-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-4-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-4-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-5-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-5-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-6-and-click-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-6-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-child-1-expected.txt:
* LayoutTests/platform/ios/fast/html/details-remove-summary-child-2-expected.txt:
* LayoutTests/platform/ios/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/ios/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/ios/fast/html/details-writing-mode-expected.txt:
* LayoutTests/platform/ios/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-no-summary4-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk1/fast/html/details-replace-summary-child-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk2/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk2/fast/html/details-replace-summary-child-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia-wk2/fast/html/details-replace-text-expected.txt: Removed.
* LayoutTests/platform/mac-sequoia/fast/html/details-no-summary4-expected.txt: Renamed from LayoutTests/platform/mac-sequoia-wk2/fast/html/details-no-summary4-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-open-javascript-expected.txt: Renamed from LayoutTests/platform/mac-sequoia-wk2/fast/html/details-open-javascript-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-open2-expected.txt: Renamed from LayoutTests/platform/mac-sequoia-wk2/fast/html/details-open2-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-open4-expected.txt: Renamed from LayoutTests/platform/mac-sonoma-wk1/fast/html/details-open4-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-replace-summary-child-expected.txt: Renamed from LayoutTests/platform/mac-sonoma-wk1/fast/html/details-replace-summary-child-expected.txt.
* LayoutTests/platform/mac-sequoia/fast/html/details-replace-text-expected.txt: Renamed from LayoutTests/platform/mac-sonoma-wk1/fast/html/details-replace-text-expected.txt.
* LayoutTests/platform/mac-sonoma-wk1/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma-wk1/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-sonoma/fast/html/details-marker-style-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/fast/html/details-marker-style-mixed-expected.txt: Added.
* LayoutTests/platform/mac-sonoma/fast/html/details-no-summary2-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-no-summary2-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-no-summary4-expected.txt: Renamed from LayoutTests/platform/mac-sonoma-wk1/fast/html/details-no-summary4-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-remove-summary-1-and-click-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-remove-summary-1-and-click-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-remove-summary-4-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-remove-summary-4-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-remove-summary-child-2-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-remove-summary-child-2-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-replace-text-expected.txt: Renamed from LayoutTests/platform/mac-sequoia-wk1/fast/html/details-replace-text-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-writing-mode-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-writing-mode-mixed-expected.txt.
* LayoutTests/platform/mac-sonoma/fast/html/details-writing-mode-mixed-expected.txt: Copied from LayoutTests/platform/mac/fast/html/details-writing-mode-mixed-expected.txt.
* LayoutTests/platform/mac-wk1/fast/html/details-no-summary4-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-replace-summary-child-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/details-replace-text-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/fast/html/text-field-input-types-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-no-summary4-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-replace-summary-child-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/details-replace-text-expected.txt: Removed.
* LayoutTests/platform/mac-wk2/fast/html/text-field-input-types-expected.txt: Removed.
* LayoutTests/platform/mac/fast/css-generated-content/details-summary-before-after-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-details-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-details-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-1-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-10-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-10-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-2-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-3-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-3-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-4-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-5-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-5-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-6-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-6-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-7-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-7-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-8-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-8-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-9-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-9-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-add-summary-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-marker-style-expected.txt:
* LayoutTests/platform/mac/fast/html/details-marker-style-mixed-expected.txt:
* LayoutTests/platform/mac/fast/html/details-nested-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-nested-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary3-expected.txt:
* LayoutTests/platform/mac/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open3-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open5-expected.txt:
* LayoutTests/platform/mac/fast/html/details-open6-expected.txt:
* LayoutTests/platform/mac/fast/html/details-position-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-1-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-2-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-3-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-3-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-4-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-4-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-5-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-5-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-6-and-click-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-6-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-child-1-expected.txt:
* LayoutTests/platform/mac/fast/html/details-remove-summary-child-2-expected.txt:
* LayoutTests/platform/mac/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/mac/fast/html/details-replace-text-expected.txt:
* LayoutTests/platform/mac/fast/html/details-writing-mode-expected.txt:
* LayoutTests/platform/mac/fast/html/details-writing-mode-mixed-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-no-summary4-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-open-javascript-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-open2-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-open4-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-replace-summary-child-expected.txt:
* LayoutTests/platform/wpe/fast/html/details-replace-text-expected.txt:
* Source/WTF/wtf/unicode/CharacterNames.h:
* Source/WebCore/css/CSSCounterStyle.cpp:
(WebCore::CSSCounterStyle::counterForSystemDisclosureClosed):
(WebCore::CSSCounterStyle::counterForSystemDisclosureOpen):

Canonical link: https://commits.webkit.org/299530@main
